pkg/otp/otp.go

```diff
@@ -31,12 +33,10 @@ func Calculate(name string, sec gopass.Secret) (twofactor.OTP, string, error) {
 	if otpURL != "" {
 		debug.Log("found otpauth url: %s", out.Secret(otpURL))
 
-		return twofactor.FromURL(otpURL) //nolint:wrapcheck
+		return otp.NewKeyFromURL(otpURL) //nolint:wrapcheck
```

Comment:
My understanding it that this supports both TOTP and HOTP:
 https://github.com/pquerna/otp/blob/60112ee2a95553a491c22956619ef30260ec93e8/otp.go#L55

Reply:
Both should work, yes.